tools/templates/vfvs_run.py

```diff
@@ -267,19 +400,175 @@ def process_ligand(task):
 
     try:
         cmd = program_runstring_array(task)
+        
+        if(task['program'] == "plants"):
+            temp_file = cmd[-1]
+            os.system('cp {} {}'.format(task['config_path'], temp_file))
+            with open(temp_file, 'a+') as f: 
+                f.writelines('ligand_file {}\n'.format(task['ligand_path']))
+                f.writelines('output_dir {}\n'.format(task['output_path'])) 
+                
+        if(task['program'] == "M-Dock"):
+            os.system('cp ./mdock_dock.mol2 {}'.format(task['output_path']))
+            os.system('rm ./mdock_dock.mol2')
+        
+        if(task['program'] == 'LigandFit'): 
+            os.system('rm -rf PDS')
```

Comment:
Similar to the comment above, the docking hasn't happened yet. So PDS probably doesn't exist, not does LigandFit_run_1_

Reply:
Likely a bug will double chech. I think I might have just added the code in the wrong block.